### ISSUE TITLE
第36回(OpsJAWSクロスイベント) - 定期的に集客する

### ISSUE THREAD

**Issue by PharaohKJ** [jaws-ug-kanazawa/jaws-ug-kanazawa.github.io#146]
- [x] JAWS-UG 金沢 slack
- [x] JAWS-UG 本家 slack
- [x] JAWS-UG 本家 slack \#jawsug-chubu
- [x] twitter(個人アカウントでよい)
- [x] facebook(公式ページのシェアなどでお願いします)
